### PR TITLE
Remove error on empty normalized word

### PR DESCRIPTION
**Pull request by KaleyKrim**
https://github.com/shinonomekazan/bad-word-detector/issues/8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bad-word-detector",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (KaleyKrim) [inline package.json:3]: Already published

File: src/tests/test.ts [hunk 2]
```diff
@@ -168,7 +171,8 @@ const testUnNormalizedBadWords = [
 	"PASSIONFRUIT",
 	"マロン",
 	"まろん",
-	"blueberry"
+	"blueberry",
+	"🌱🌱",
```
Comment (KaleyKrim) [inline src/tests/test.ts:175]: isBad = true

File: src/tests/test.ts [hunk 3]
```diff
@@ -178,7 +182,8 @@ const testUnNormalizedOkWords = [
 	"ｐássioｎｆｒｕｉｔ",
 	"passionfruit",
 	"BLUEBERRY",
-	"iloveblueberry"
+	"iloveblueberry",
+	"🌱🌱🌱",
```
Comment (KaleyKrim) [inline src/tests/test.ts:186]: isBad = false

**Review by tsugehara — APPROVED** [review]
LGTM!

**Pull request merged**